tests/ui-toml/float_cmp_named_constants/test.rs

```diff
@@ -0,0 +1,18 @@
+//@no-rustfix
+
+#![deny(clippy::float_cmp)]
+
+fn main() {
+    {
```

Comment:
Could you move the `const C` outside of the `main` function and also move both `f` functions out too please? Having nested levels like this make the code reading a bit more complicated than it could be.

Reply:
Nested blocks both move the code for each test closer together (as in, everything needed for each test is all in one spot). and prevents name collisions between tests.